index.md

```diff
@@ -4,15 +4,10 @@ root: .
 ---
 
 The Unix shell has been around longer than most of its users have been alive.
-It has survived so long because it's a powerful tool
-that allows people to do complex things with just a few keystrokes.
-More importantly,
-it helps them combine existing programs in new ways
-and automate repetitive tasks
-so they aren't typing the same things over and over again.
-Use of the shell is fundamental to using a wide range of other powerful tools
-and computing resources (including "high-performance computing" supercomputers).
-These lessons will start you on a path towards using these resources effectively.
+It has survived because it's a powerful tool that allows users to perform complex and powerful tasks, often with just a few keystrokes or lines of code.
+It not only helps users automate repetitive tasks, but easily combine smaller tasks into larger, more powerful workflows.
```

Comment:
"but -> and" I think

Reply:
I would have to change the structure of the sentence a little more because of the "not only" construction. I'll change it to: "It helps users automate repetitive tasks and easily combine smaller tasks into larger, more powerful workflows." if that works.

Reply:
I just pushed up the change.

Reply:
Yes absolutely.